### PR TITLE
Fix: when using custom table the choice list was still populated from core tags table

### PR DESCRIPTION
**Pull request by jaanusnurmoja**
tags.js uses now options=com_fabrik url (instead com_tags) but it shouldn't change anything as the default table is still #__tags

### CODE REVIEW

File: plugins/fabrik_element/tags/tags.php [hunk 1]
```diff
@@ -326,10 +328,26 @@ protected function buildQuery($data = array(), $incWhere = true, $opts = array()
 		$query->select('DISTINCT(t.id) AS value,' . $db->qn('title') . ' AS text')
 		->from($db->qn($join->table_join) . ' AS ' . $db->qn($join->table_join_alias))
 		->join('LEFT', $this->getDbName() . ' AS t ON t.id = ' . $db->qn($join->table_join_alias . '.' . $join->table_key));
+		$db->setQuery($query);
+		$query->opts = $db->loadObjectList();
 
 		return $query;
 	}
 
+	public function allTagsJSON()
```
Comment (pollen8) [inline plugins/fabrik_element/tags/tags.php]: could you add a PHPdoc block comment to this function please?

File: plugins/fabrik_element/tags/tags.php [hunk 2]
```diff
@@ -60,7 +60,10 @@ public function elementJavascript($repeatCounter)
 		$opts = $this->getElementJSOptions($repeatCounter);
 		$opts->rowid = $this->getFormModel()->getRowId();
 		$opts->id = $this->id;
-
+		$opts->allTags = $this->allTagsJSON();
+		$opts->formid = $this->getFormModel()->id;
+		$opts->listid = $this->getListModel()->getId();
+		$opts->url = $this->tagListURL();
 		return array('FbTags', $id, $opts);
```
Comment (pollen8) [inline plugins/fabrik_element/tags/tags.php]: codestyle: best to add a new line before the return statement for

File: plugins/fabrik_element/tags/tags.php [hunk 3]
```diff
@@ -60,7 +60,10 @@ public function elementJavascript($repeatCounter)
 		$opts = $this->getElementJSOptions($repeatCounter);
 		$opts->rowid = $this->getFormModel()->getRowId();
 		$opts->id = $this->id;
-
+		$opts->allTags = $this->allTagsJSON();
+		$opts->formid = $this->getFormModel()->id;
+		$opts->listid = $this->getListModel()->getId();
+		$opts->url = $this->tagListURL();
```
Comment (pollen8) [inline plugins/fabrik_element/tags/tags.php]: code style: best to add a line return before the return statement

**Comment by jaanusnurmoja** [comment]
@pollen8 Done with some cleanup

**Comment by jaanusnurmoja** [comment]
Reason?

**Comment by jaanusnurmoja** [comment]
@cheesegrits OK,  changed the fields.xml - removed tags_dbname_desc element and moved the description to the tags_dbname, but also added more information to the label to be seen immediately. + some more works.

**Comment by jaanusnurmoja** [comment]
P.S. Please, can anyone make a minified js as tags.js is also changed

**Pull request merged**